crypto/evp/m_sigver.c

```diff
@@ -191,6 +192,11 @@ static int do_sigver_init(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
              */
             ctx->digest = ctx->reqdigest = ctx->fetched_digest =
                 EVP_MD_fetch(locpctx->libctx, mdname, props);
+            if (ctx->digest == NULL) {
```

Comment:
Should this be marked with a comment such as
/* legacy code support for engines */

Reply:
Done. Thanks!